packages/components/organisms/f-checkout/src/components/Error.vue

```diff
@@ -22,23 +21,75 @@
             data-test-id="checkout-error-page-description">
             {{ $t(`errorMessages.${errorFormType}.description`) }}
         </p>
+
+        <f-button
+            :class="$style['c-checkout-error-button']"
+            button-size="large"
+            button-type="primary"
+            is-full-width
+            data-test-id="error-page-redirect-to-menu-button"
+            @click.native="redirectToMenu">
+            {{ $t(`errorMessages.${errorFormType}.buttonText`) }}
+        </f-button>
     </card>
 </template>
 
 <script>
+import { mapState } from 'vuex';
 import Card from '@justeat/f-card';
+import FButton from '@justeat/f-button';
 import SadBagIconDecorator from '../assets/images/jet-sad-bag.svg';
+import '@justeat/f-button/dist/f-button.css';
 import '@justeat/f-card/dist/f-card.css';
+import { VUEX_CHECKOUT_MODULE } from '../constants';
+import loggerMixin from '../mixins/logger.mixin';
 
 export default {
     components: {
         Card,
+        FButton,
         SadBagIconDecorator
     },
+    mixins: [
+        loggerMixin
+    ],
     props: {
         errorFormType: {
             type: String,
             required: true
+        },
+        restaurantMenuPageUrl: {
+            type: String,
+            default: ''
+        }
+    },
+
+    computed: {
+        ...mapState(VUEX_CHECKOUT_MODULE, [
+            'basket'
+        ])
+    },
+
+    mounted () {
+        if (this.$logger) {
+            this.logInvoker({
+                message: 'Consumer Checkout Error Page',
+                data: {},
+                logMethod: this.$logger.logWarn
+            });
+        }
+    },
+
+    methods: {
+        redirectToMenu () {
```

Comment:
I see we're always deleting the basket cookie here if it exists, but we should only delete the basket cookie when we get a 403 error, not in other cases.

Reply:
Will check but according to Maryia this was working as expect, only the tests and logging was left to do so I assumed this was fine. Ill re-check tho.

Reply:
@SachaZvetelman @kevinrodrigues if user gets the error page and we advice him to return to the menu page, we want it to be a fresh menu page without saved basket, so I added a check that if there is a basket id in cookies than remove it, if not just redirect to menu page.

Reply:
I re-checked the ACs and it does say delete the cookie when there's a 403? Any other error would remove the basket when going back to menu. Let me know if we should specifically check for the 403 here first.

Reply:
I amended the condition with a check that errorFormType is 403 page